src/vm/unixlowmem.h

```diff
@@ -0,0 +1,23 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+// ===========================================================================
+
+#ifndef _UNIXLOWMEM_H_
+#define _UNIXLOWMEM_H_
+
+#ifdef FEATURE_PAL
+class UnixLowMemoryDetector
+{
+    size_t m_szLowMemoryLimitBytes;
```

Comment:
A nit - could you please remove the "sz" prefix from all locals and this member? Looking at our sources, I don't see it being used anywhere, so it would be nice to keep the convention.